tr/SUMMARY.md

```diff
@@ -1,26 +1,51 @@
 # Özet
 
-* [Giriş](README.md)
-* [Kurulum](installation/README.md)
-* [İnternet nasıl çalışır](how_the_internet_works/README.md)
-* [Komut satırına giriş](intro_to_command_line/README.md)
-* [Python kurulumu](python_installation/README.md)
-* [Kod editörü](code_editor/README.md)
-* [Python'a giriş](python_introduction/README.md)
-* [Django nedir?](django/README.md)
-* [Django kurulumu](django_installation/README.md)
-* [İlk Django projen!](django_start_project/README.md)
-* [Django modelleri](django_models/README.md)
-* [Django admin](django_admin/README.md)
-* [Yayına alın!](deploy/README.md)
-* [Django url'leri](django_urls/README.md)
-* [Django views - yaratma zamanı geldi!](django_views/README.md)
-* [HTML'ye giriş](html/README.md)
-* [Django ORM ve QuerySets (Sorgu Setleri)](django_orm/README.md)
-* [Template içerisinde dinamik veri](dynamic_data_in_templates/README.md)
-* [Django template](django_templates/README.md)
-* [CSS - sayfanı güzelleştir](css/README.md)
-* [Template genişletmek](template_extending/README.md)
-* [Uygulamanı genişlet](extend_your_application/README.md)
-* [Django formları](django_forms/README.md)
-* [Sırada ne var?](whats_next/README.md)
+*   [Giriş][1]
```

Comment:
Instead of:

```markdown
[Giriş][1]

[1]: README.md
```

it should be:

```markdown
[Giriş](README.md)
```

You have to do it manually, because crowding generates this file in incorrect format :/